x-pack/plugin/core/src/main/resources/synthetics-mappings.json

```diff
@@ -0,0 +1,57 @@
+{
+  "template": {
+    "mappings": {
+      "dynamic_templates": [
+        {
+          "strings_as_keyword": {
+            "mapping": {
+              "ignore_above": 1024,
+              "type": "keyword"
+            },
+            "match_mapping_type": "string"
+          }
+        }
+      ],
+      "date_detection": false,
+      "properties": {
+        "@timestamp": {
+          "type": "date"
+        },
+        "data_stream": {
+          "properties": {
+            "type": {
+              "type": "constant_keyword",
+              "value": "synthetics"
+            },
+            "dataset": {
+              "type": "constant_keyword"
+            },
+            "namespace": {
+              "type": "constant_keyword"
+            }
+          }
+        },
+        "ecs": {
+          "properties": {
+            "version": {
+              "ignore_above": 1024,
+              "type": "keyword"
+            }
+          }
+        },
+        "host": {
```

Comment:
I'm actually wondering if we can drop this field. We don't use `host.*` but rather `observer.*` in synthetics. It's not a big deal (we could just ignore it), but it isn't going to be used.

Reply:
It makes sense to drop it if we're not going to use it. I will remove it

Reply:
@dakrone The `host.ip` is here for safety reason that nobody ships data with `host` as keyword, so I would rather keep it. And to be double save, in the synthetics case it probably makes sense then to add `observer.ip`.

Reply:
Alright, since this was already merged I'll open a different PR to re-add both of these fields

Reply:
Opened https://github.com/elastic/elasticsearch/pull/62412 for this.